applications/nublado2/values-usdfprod.yaml

```diff
@@ -94,8 +94,6 @@ config:
 #   - name: fs-ddn-sdf-group-rubin
 #     mountPath: /instrument
 #     subPath: lsstdata/offline/instrument
-    - name: fs-nfs
```

Comment:
Need to remove lines 64 and 65 as well?

Reply:
Thanks!

Reply:
Done, and removed some commented-out mountpoints